.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+jobs:
+  build:
+    docker:
+    - image: circleci/node:8.11.1
+    steps:
+    - checkout
+    - run:
+        name: Code Linting
+        command: |
+          yarn && yarn lint
+  client-test:
+    docker:
+    - image: circleci/node:8.11.1
+    steps:
+    - checkout
+    - run:
+        name: run client tests
+        command: |
```

Comment:
you can use workspace directory over here, this will be helpful if you use multiple commands.